pkg/apis/servicecatalog/types.go

```diff
@@ -278,7 +279,9 @@ type BindingSpec struct {
 	// should be injected with the results of the binding.  Immutable.
 	AppLabelSelector metav1.LabelSelector
 
-	Parameters map[string]runtime.Object
+	// Parameters is a YAML representation of the properties to be
+	// passed to the underlying broker.
+	Parameters string `json:"parameters"`
```

Comment:
`interface{}` is not handled in the API machinery.  Fields that hold raw serialized JSON/YAML data should be `runtime.RawExtension` in the versioned API and `runtime.Object` in the internal API.